vulture_os/gui/static/js/listener.js

```diff
@@ -165,7 +165,13 @@ $(function() {
     });
 
     /* Re-initialize select2 objects */
-    $('.select2').select2();
+    let selects = Array.prototype.slice.call(document.querySelectorAll('.select2'));
+    selects.forEach(function(html) {
+      $(html).select2();
+      if (html.attributes['data-original-title']) {
+        $(html.nextElementSibling).tooltip({title: html.attributes['data-original-title'].value, placement: "bottom"});
+      }
+    });
```

Comment:
is that linked to the main change?

Reply:
Good catch, the change has already been included in this other PR: #504.
I will delete this change here.